docs/UserGuide.md

```diff
@@ -118,6 +120,34 @@ Example:
 * `task A0123456Z`
     * Lists out the tasks that student (A0123456Z) has.
     
+### Marking an undone task as done for a particular student: `mark`
+
+Marks a specific undone task as done for a particular student.
+
+Format : `mark n/STUDENT_NAME UNDONE_TASK_INDEX` (or) `mark i/STUDENT_ID UNDONE_TASK_INDEX`
+
+* The undone task corresponding to the index or the particular student will be marked as done in the TAPA.
```

Comment:
Might want to include what the user should expect if an invalid index is supplied (maybe an error message or smt?)

Reply:
Added what user should expect if an invalid index is supplied (when out of bounds and when task is already marked